Src/PerformanceCollector/Unit.Tests/PerformanceCollectorTestBase.cs

```diff
@@ -89,18 +82,14 @@ public void PerformanceCollectorRefreshCountersTest()
             Assert.AreEqual(collector.PerformanceCounters.Last().InstanceName, "_Total123afadfdsdf");
         }
 
-        [TestMethod]
-        [TestCategory("RequiresPerformanceCounters")]
-        public void PerformanceCollectorBadStateTest()
+        internal void PerformanceCollectorBadStateTest(IPerformanceCollector collector)
```

Comment:
This is a base class that is used by two collectors, which is why I have moved the code out.